lib/evss/pciu_address/response_strategy.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+require 'common/models/concerns/cache_aside'
+
+module EVSS
+  module PCIUAddress
+    class ResponseStrategy < Common::RedisStore
```

Comment:
What's the rationale for calling this "Strategy"?

Reply:
Strategy is the name of the design pattern that encapsulates two (or more) algorithms that output a similar result, in this case delivering the response via the service or cache.